packages/mobile-first-components/src/components/TabButton.tsx

```diff
@@ -12,7 +11,7 @@ const TabButton = (props: {
   title: string;
   onPress: Function;
 }) => (
-  <TouchableOpacity onPress={props.onPress} style={{ flex: 1 }}>
+  <button onClick={() => props.onPress} style={{ flex: 1 }}>
```

Comment:
yes

Reply:
I wonder if we might want to have a directory of really basic primitives, like `Button` instead of using the DOM button directly. That way when / if we add back react native support, it would just be a matter of overriding a handful of simple primitives.

```js
const Button = props => <button {...props}>{props.children}</button>;

const View = props => <div {...props}>{props.children}</div>;

const Text = props => <span {...props}>{props.children}</span>;
```

For example